API/Services/Tasks/Scanner/Parser/Parser.cs

```diff
@@ -25,7 +25,7 @@ public static class Parser
 
     public static readonly TimeSpan RegexTimeout = TimeSpan.FromMilliseconds(500);
 
-    public const string ImageFileExtensions = @"^(\.png|\.jpeg|\.jpg|\.webp|\.gif|\.avif)"; // Don't forget to update CoverChooser
+    public const string ImageFileExtensions = @"^(\.png|\.jpeg|\.jpg|\.webp|\.gif|\.avif|\.jxl|\.heif|\.heic|\.j2k|\.jp2)"; // Don't forget to update CoverChooser
```

Comment:
Looks like you forgot to update CoverChooser component on the UI

Reply:
I wasn't sure on this, Cover is converted to jpg in case of the new formats.